Source/ARTConnection.h

```diff
@@ -64,6 +55,12 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void)ping:(ARTCallback)callback;
 
+/**
+ * The recovery key string can be used by another client to recover this connection's state in the recover client options property. See [connection state recover options](https://ably.com/docs/realtime/connection#connection-state-recover-options) for more information.
```

Comment:
The doc for this one was taken from previous recoveryKey property - I added the docs about when this can be nil here 0eb201495aeeadaa3926dfaf084e5d894523fdfe